### ISSUE TITLE
5 test passed the Nondex with 100 runs for the apolloconfig/apollo project in the latest version

### ISSUE THREAD

**Issue by bsknblc** [TestingResearchIllinois/idoft#586]
In the links below there are 5 tests that past the Nondex with 100 runs. In the Idoft database, they don't have any PR opened for them but they still pass the Nondex.
4 tests passed in this build https://github.com/bsknblc/apollo/actions/runs/2455856038
1 test passed in this build https://github.com/bsknblc/apollo/actions/runs/2455662642

**Comment by winglam** [comment]
Thanks for adding this issue report. Can you please add a link to this issue report to the "Notes" column of the tests that are relevant to this issue report? (So that the next person that debugs those tests knows of this issue report.)

**Comment by Auzel** [comment]
@bsknblc I believe you may have unintentionally switched the results of the builds. For example, the build referenced by your first link (https://github.com/bsknblc/apollo/actions/runs/2455856038) has one passed test, and the build referenced by your second link (https://github.com/bsknblc/apollo/actions/runs/2455662642) has 4 passed test.

I'm not sure which commit you are working from, but the 4 tests have a PR opened, which is cited in pr-data.csv.
```testAssembleLongPollRefreshUrl,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4095,
testAssembleLongPollRefreshUrlWithMultipleNamespaces,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4095,
testAssembleQueryConfigUrl,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4083,
testGetPropertyNames,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4082
```

For the sole test that doesn't have a PR opened (testWhenHasReleaseMsgAndHasRepeatMsg), it is currently being looked at.

**Comment by bsknblc** [comment]
Hello, yes it looks like I switched the URLs. It was a bit late when I was preparing this text, sorry for the confusion.

I checked the tests in the `pr-data.csv` file and it was from my forked repository which I forked a year ago. There wasn't a PR opened back then so again I am sorry for the confusion. Should I add a note for the `testWhenHasReleaseMsgAndHasRepeatMsg` test or close this issue because it is being looked at.
